Index: src/masks.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>def get_mask_card_number(card_number):\r\n    visible_digits = 6\r\n    masked_number = (\r\n        card_number[:visible_digits]\r\n        + \" \"\r\n        + \" \".join(\r\n            [\"**\" if i < visible_digits + 2 else \"****\" for i in range(visible_digits, len(card_number) - 4, 4)]\r\n        )\r\n        + \" \"\r\n        + card_number[-4:]\r\n    )\r\n    return masked_number\r\n\r\n\r\ndef get_mask_account(account_number):\r\n    visible_digits = 4\r\n    masked_number = \"**\" + account_number[-visible_digits:]\r\n    return masked_number\r\n\r\n\r\n# изменение в Git 1\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/masks.py b/src/masks.py
--- a/src/masks.py	(revision 55c02a33de7d23425485a81422c40493073af19f)
+++ b/src/masks.py	(date 1718853601087)
@@ -1,17 +1,10 @@
 def get_mask_card_number(card_number):
-    visible_digits = 6
-    masked_number = (
-        card_number[:visible_digits]
-        + " "
-        + " ".join(
-            ["**" if i < visible_digits + 2 else "****" for i in range(visible_digits, len(card_number) - 4, 4)]
-        )
-        + " "
-        + card_number[-4:]
-    )
-    return masked_number
+     masked_number = card_number[:4] + ' '+card_number[4:6]+'**' + ' **** ' + card_number[-4:]
+     return masked_number
 
 
+
+
 def get_mask_account(account_number):
     visible_digits = 4
     masked_number = "**" + account_number[-visible_digits:]
